clifford/test/test_function_cache.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+from numba import generated_jit
+import pytest
+
+
+@generated_jit(cache=True)
```

Comment:
This either need to use `_numba_utils`, or be disabled on runs when jitting is disabled